src/commands/deployments/create.ts

```diff
@@ -17,19 +17,22 @@ export const createDeployment = async (
 ): Promise<Deployment> => {
 	const client = getDeploymentApiClient(args.token);
 	try {
-		const lastDeployment = await client.getLatestDeploymentDeprecated({
-			appId: args.appId,
-		});
-
+		let lastDeployment: Deployment | undefined;
 		if (
-			lastDeployment === undefined &&
-			(args.buildId === undefined ||
-				args.roomsPerProcess === undefined ||
-				args.planName === undefined ||
-				args.transportType === undefined ||
-				args.containerPort === undefined)
+			args.buildId === undefined ||
+			args.roomsPerProcess === undefined ||
+			args.planName === undefined ||
+			args.transportType === undefined ||
+			args.containerPort === undefined
 		) {
-			throw new Error("All args must be present for the initial deployment");
+			console.log("Some args missing, copying values from the last deployment");
+			try {
+				lastDeployment = await client.getLatestDeploymentDeprecated({
+					appId: args.appId,
+				});
+			} catch (e) {
+				throw new Error("No previous deployment found");
```

Comment:
nit: I would write something like "buildId, roomsPerProcess,...etc must be defined if not previous deployment exists"

Reply:
Did we want to log this to console? Not sure how unhandled exceptions are treated, we only seem to catch `ResponseError`

Reply:
In the cli i believe this will get logged to the console